.github/workflows/artifacts-build.yml

```diff
@@ -0,0 +1,75 @@
+# This file is part of ICU4X. For terms of use, please see the file
+# called LICENSE at the top level of the ICU4X source tree
+# (online at: https://github.com/unicode-org/icu4x/blob/main/LICENSE ).
+
+##### README #####
+#
```

Comment:
I know this is handy to have here when you , but would the contents here (and the warning message) be better moved to actual docs, https://github.com/unicode-org/icu4x/tree/main/docs  since it's presumably also part of the processes?

Reply:
I considered that, but in this case I thought it would be better to have the documentation very close in the code to where it is used. I can move it if you or others feel strongly.